src/probnum/randvars/_arithmetic.py

```diff
@@ -288,8 +291,13 @@ def _matmul_normal_constant(norm_rv: _Normal, constant_rv: _Constant) -> _Normal
         # because of performance configurations: currently, there is no way of switching
         # the Cholesky updates off, which might affect (large, potentially sparse) covariance matrices
         # of matrix-variate Normal RVs. See Issue #335.
+        if constant_rv.support.ndim == 1:
+            constant_rv_support = constant_rv.support[:, None]
+        else:
+            constant_rv_support = constant_rv.support
         cov_update = _linear_operators.Kronecker(
-            _linear_operators.Identity(constant_rv.shape[0]), constant_rv.support
+            _linear_operators.Identity(norm_rv.shape[0]),
+            constant_rv_support,
         )
 
         return _Normal(
```

Comment:
```suggestion
            _linear_operators.Identity(norm_rv.shape[0]),
            constant_rv_support.T,
        )
        
        return _Normal(
            mean=norm_rv.mean @ constant_rv.support,
            cov=cov_update @ (norm_rv.cov @ cov_update.T),
        )
```

IMHO, this is easier to understand from the defining property of the Kronecker product:

```
X @ v == I @ X @ v == I @ X @ (v.T).T == kron(I, v.T) @ vec(X)
```

- [ ] if we change this, the tests should probably be adjusted, too

Note that the suggestion can't be applied directly, since I can not comment past line 303